app/lib/theme/subthemes/text_theme.dart

```diff
@@ -50,7 +51,7 @@ class FlexTextTheme {
     bodySmall: textStyle.copyWith(
       fontSize: 14.0,
       fontWeight: FontWeight.normal,
-      color: FlexAppColorScheme.lightScheme.onSurface.withOpacity(0.5),
+      color: FlexAppColorScheme.lightScheme.onSurface.adjustOpacity(0.5),
```

Comment:
Just thinking out loud based on our discussion, this should work 95% of the time (using `onSurface`) but we may need to look at adding a `textPrimary` or something to the `FlexAppColorScheme`.